src/metalperformanceshaders.cs

```diff
@@ -1028,7 +1028,7 @@ interface MPSCnnConvolution {
 		[Export ("neuronParameterB")]
 		float NeuronParameterB { get; }
 
-		[TV (11, 0), iOS (11, 0)]
+		[TV (11,2), iOS (11,2), Mac (10,13,2, onlyOn64: true)]
```

Comment:
really ? the header diffs were cleared as being only noise

Reply:
Yup, intro on device complains if iOS < 11.2

Reply:
```
***** ApiSelectorTest.InstanceMethods
[FAIL] Selector not found for MetalPerformanceShaders.MPSCnnConvolution : neuronParameterC
***** ApiSelectorTest.Protocols
***** ApiSelectorTest.StaticMethods
```
This is in macOS 10.13.1

Reply:
@spouliot  and for the sake of completeness Apple docs agrees https://developer.apple.com/documentation/metalperformanceshaders/mpscnnconvolution/2935626-neuronparameterc?language=objc